packages/hoppscotch-selfhost-desktop/src/platform/interceptors/native/index.ts

```diff
@@ -129,7 +130,8 @@ async function processBody(axiosReq: AxiosRequestConfig): Promise<BodyDef | null
           value: {
             File: {
               filename: value.name,
-              data: new Uint8Array(await value.arrayBuffer())
+              data: Array.from(new Uint8Array(await value.arrayBuffer())),
+              mime: value.type,
```

Comment:
If the browser/system doesn't know the Mime Type, `value.type` can be an empty string so we should avoid putting in the mime type in that situation.
See: https://arc.net/l/quote/thjiuzih

Reply:
Yes, you're right!
By [rfc2046](https://www.ietf.org/rfc/rfc2046.txt) if mime type is not recognizable you should use `"application/octet-stream"` content type